plugin.go

```diff
@@ -169,7 +175,14 @@ func CopyTfEnv() {
 	}
 }
 
-func assumeRole(roleArn string) {
+func assumeRole(roleArn string) bool {
+	awsTokens := []string{"AWS_ACCESS_KEY_ID", "AWS_SECRET_ACCESS_KEY", "AWS_SESSION_TOKEN"}
```

Comment:
I'm guessing this is where we check if creds are already set.  Would it make more sense to put this in a function and add that to `if p.Config.RoleARN != ""` (becomes `if p.Config.RoleARN != "" && !credsSet()`) check before we call `assumeRole`?

Hope that made sense ... 😖

Reply:
Make sense for me, will do!